lib/rubygems/specification.rb

```diff
@@ -2698,7 +2698,7 @@ def validate packaging = true
 
     unless specification_version.is_a?(Integer)
       raise Gem::InvalidSpecificationException,
-            'specification_version must be a Integer (did you mean version?)'
+            'specification_version must be an Integer (did you mean version?)'
     end
```

Comment:
Please report to https://github.com/rubygems/rubygems, it will be overwritten otherwise.